services/ui-src/src/components/cards/EntityCard/EntityCard.test.tsx

```diff
@@ -46,23 +44,10 @@ const mockUnfinishedEntityData = {
   standardType: "mock-standardType",
 };
 
-const EntityCardComponent = (
-  <EntityCard
-    entity={mockEntity}
-    entityType={mockEntityType}
-    formattedEntityData={mockFormattedEntityData}
-    verbiage={mockModalDrawerReportPageJson.verbiage}
-    openAddEditEntityModal={openAddEditEntityModal}
-    openDeleteEntityModal={openDeleteEntityModal}
-    openDrawer={mockOpenDrawer}
-    data-testid="mock-entity-card"
-  />
-);
-
 const UnfinishedEntityCardComponent = (
   <EntityCard
     entity={mockEntity}
-    entityType={mockEntityType}
+    entityType={"mock-entity-type"}
```

Comment:
Nit: Just strings'll do. 
```suggestion
    entityType="mock-entity-type"
```